src/Umbraco.Cms.ManagementApi/Controllers/MananagementApiBaseController.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.ManagementApi.Filters;
+
+namespace Umbraco.Cms.ManagementApi.Controllers;
+
+[SystemTextJsonConfiguration]
+public class MananagementApiBaseController : Controller
```

Comment:
Suggest renaming to `MananagementApiControllerBase` to follow the general naming convention